src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider(name = "DataHtmlTitleTest_001")
+  private Object[][] DataHtmlTitleTest_001() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)